README.md

```diff
@@ -16,108 +18,172 @@ It is meant to be used when building transactions that upgrade the implementatio
 
 If you would like to contribute a new chain or superchain, please see our [contributing guide](./CONTRIBUTING.md).
```

Comment:
Is this callout still necessary, given that the instructions are now directly in this readme?

Reply:
nope, just missed that, thanks!